src/gui/src/components/data-table/table-filter-search.tsx

```diff
@@ -45,7 +47,8 @@ export const TableFilterSearch = ({
   }, [])
 
   return (
-    <div className="relative flex w-[384px] items-center">
+    <div
+      className={`relative flex w-[384px] items-center ${className}`}>
```

Comment:
```suggestion
      className={`relative flex w-96 items-center ${className}`}>
```

oh, `w-96` is that exactly same value! 😄